maquetado/scss/form.scss

```diff
@@ -0,0 +1,59 @@
+.info-container {
+  background-color: $wild-sand;
+  justify-content: space-between;
```

Comment:
You don't need this. The space is already being determined by the inner elements and the container's padding